src/OrchardCore/OrchardCore.ContentManagement.Abstractions/Handlers/IContentPartHandler.cs

```diff
@@ -19,8 +20,15 @@ public interface IContentPartHandler
         Task ImportedAsync(ImportContentContext context, ContentPart part);
         Task UpdatingAsync(UpdateContentContext context, ContentPart part);
         Task UpdatedAsync(UpdateContentContext context, ContentPart part);
+        Task ValidatingAsync(ValidatePartContentContext context, ContentPart part);
+        Task ValidatedAsync(ValidatePartContentContext context, ContentPart part);
+
+        [Obsolete("This method will be removed in future releases. Instead, use ValidatingAsync(ValidatePartContentContext context, ContentPart part)")]
```

Comment:
Doesn't like an idea to add `Obsolete` for no reason

Reply:
@ns8482e 
These methods will add no value since they are replaced by another method. I kept them to not introduce a breaking change as you saw happened in IDisplayManager

Reply:
It's  better you find a way to use same method, than introducing new one

Reply:
@ns8482e I made the requested change. Let me know if you see something else that needs updating.